### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by 1010sachin**

### CODE REVIEW

**Comment by niravpatel27** [comment]
@1010sachin You will have to mock the directory. Try creating a dummy directory(dont use actual directory name) and remove it once you are done.

**Comment by niravpatel27** [comment]
Looks good!

File: get/cluster_test.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+	if expected != err.Error() {
+		t.Error(err)
+	}
+
+}
+
+func TestUnidentifiedClusterManagerNonInteractiveMode(t *testing.T){
+	viper.Set("non-interactive", true)
+	viper.Set("cluster_manager", "xyz")
```
Comment (niravpatel27) [inline get/cluster_test.go]: Also, we need to clean variables from the viper object once we are done with that test as its going affect other tests.

Comment (1010sachin) [reply get/cluster_test.go]: Okay, will add that

Comment (niravpatel27) [reply get/cluster_test.go]: I actually dont see other way in `viper` besides setting variable value to `nil` or call `viper.reset()` @nimajalali any suggestion here? Viper seems to be using `os.setEnv` internally

Comment (1010sachin) [reply get/cluster_test.go]: viper.Reset() sets the value to nil too.

**Pull request merged**